Makefile

```diff
@@ -4,7 +4,7 @@ lint:
 	golangci-lint run
 
 test:
-	go test -race -bench=$$^ -timeout=5m ./...
+	go test -v -race -bench=$$^ -timeout=5m ./...
```

Comment:
Running with `-v` reduces memory usage?

Reply:
no this is WIP. I was checking to see if there is a particular test that always causes the crash. -v prints the test names as they run.